Dockerfile

```
@@ -17,5 +17,8 @@ RUN /usr/local/bin/composer dump-autoload
 
 ENV NAAD_NAME=NAAD-1
 ENV NAAD_URL=streaming1.naad-adna.pelmorex.com
+ENV DESTINATION_URL=https://localhost/eibc/wp-json/naad/v1/alert
```

i would try to refrain from using specifics here, even though it means you won't have to create an env for this.
```suggestion
ENV DESTINATION_URL=https://localhost/wp-json/naad/v1/alert
```